tools/bin/sdc-sbcreate

```diff
@@ -118,78 +122,83 @@ function grab_default
 function grab_sysinfo
 {
     echo "  > sysinfo"
-    sysinfo > ${TEMPDIR}/sysinfo.json
+    sysinfo > "${TEMPDIR}/sysinfo.json"
 }
 
 function grab_hw_config
 {
     echo "  > hardware configuration"
-    prtdiag -v > ${TEMPDIR}/prtdiag
-    prtconf -v > ${TEMPDIR}/prtconf
-    smbios -x > ${TEMPDIR}/smbios
-    psrinfo -v > ${TEMPDIR}/psrinfo
+    prtdiag -v > "${TEMPDIR}/prtdiag"
+    prtconf -v > "${TEMPDIR}/prtconf"
+    smbios -x > "${TEMPDIR}/smbios"
+    psrinfo -v > "${TEMPDIR}/psrinfo"
 }
 
 function grab_dlinfo
 {
     echo "  > datalink info"
-    dladm show-phys -m > ${TEMPDIR}/dladm.phys
-    dladm show-vnic > ${TEMPDIR}/dladm.vnic
-    dladm show-linkprop > ${TEMPDIR}/dladm.linkprop
-    dladm show-bridge > ${TEMPDIR}/dladm.bridge
+    dladm show-phys -m > "${TEMPDIR}/dladm.phys"
+    dladm show-vnic > "${TEMPDIR}/dladm.vnic"
+    dladm show-linkprop > "${TEMPDIR}/dladm.linkprop"
+    dladm show-bridge > "${TEMPDIR}/dladm.bridge"
 }
 
 function grab_ifconfig
 {
     echo "  > network interface info"
-    ifconfig -a > ${TEMPDIR}/ifconfig.a
+    ipadm show-addr > "${TEMPDIR}/ipadm.show-addr"
 }
 
 function grab_routes
 {
     echo "  > routing table"
-    netstat -rn > ${TEMPDIR}/netstat.rn
+    netstat -rn > "${TEMPDIR}/netstat.rn"
```

Comment:
Add "-c" to this ("netstat -rnc"), so we can see /NN in the output.

Reply:
Still missing -c here.

Reply:
Fixed now.